README.md

```diff
@@ -70,6 +70,31 @@ bash scripts/setup.sh
 
 (You can also create a new AzureML workspace in an existing resource group by only specifying `TF_VAR_existing_resource_group_name` but not `TF_VAR_existing_workspace_name`.)
 
+### Option 3: Only create a new Syncer VM, and integrate exiting Azure ML workspace with existing Pachyderm cluster
+
+You will need to copy the Terraform code to a fresh new directory, and create a new Syncer VM
+
+```
+mkdir syncer1  # recommend naming this as syncer-$workspace_name
+cp terraform/*.tf syncer1
+cp -R terraform/out/ syncer1/out
```

Comment:
Up to you, but I think if someone does try to create a syncer VM pointed at a cluster they provisioned themselves, it might be helpful to them to have a comment like `# setup.sh will read syncer1/out/kubeconfig, so make sure that's present`